examples/pipelines/ccfraud/submit.py

```diff
@@ -279,6 +312,15 @@ def fl_ccfraud_basic():
             # make sure the compute corresponds to the silo
             silo_training_step.compute = silo_config.compute
 
+            # set distribution according to the number of available GPUs (1 in case of only CPU available)
+            silo_training_step.distribution.process_count_per_instance = silo_processes
+
+            # set number of instances to distribute training across
+            if hasattr(silo_config, "instance_count"):
```

Comment:
this parameter instance_count is not in the config.yaml and is not documented in ccfraud.md

could you add the parameter in config.yaml? (even commented out, with a note that explains why it's relevant) and document it explicitely in your section of ccfraud.md ?